pkgs/development/compilers/openjdk/JavaGetVersion.java

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Is there a particular reason why the updater is not written in bash or python other than dog fooding?

It makes it a lot harder for general contributions to use it.

Reply:
Two reasons:
- The update script uses https://docs.oracle.com/en/java/javase/22/docs/api/java.base/java/lang/Runtime.Version.html to parse and compare versions. It makes more sense to reuse that instead of re-implementing that logic on other languages, particularly since the Java version schema could be more complex than expected: https://openjdk.org/jeps/223
- I expect Java developers to be more interested in maintaining the updater script. I think it's fair to write the script in a language more familiar to them.